pkg/sqlite/load.go

```diff
@@ -942,6 +943,9 @@ func (s *sqlLoader) RemovePackage(packageName string) error {
 		if err != nil {
 			return err
 		}
+		if len(csvNames) == 0 {
+			logrus.WithFields(logrus.Fields{"pkg": packageName}).Warnf("package doesn't exist")
```

Comment:
Should this be an error instead of a warning?

Reply:
We should probably just return here.

Reply:
Do we need some testing if we're changing behavior here?

Reply:
Thanks for reviewing.  I'll change the warning to the error and return here.  I'll look into the test change, too.

Reply:
I think the linter error is just fixed by running `go-fmt`